spec/presenters/tslr_claim_csv_row_spec.rb

```diff
@@ -43,13 +43,35 @@
         date_of_birth,
         claim.teacher_reference_number,
         claim.national_insurance_number,
-        "Plan 2",
-        claim.email_address,
+        StudentLoans::PLAN_2.humanize,
+        "=\"#{claim.email_address}\"",
         mostly_teaching_eligible_subjects,
         claim.bank_sort_code,
         claim.bank_account_number,
         student_loan_repayment_amount,
       ])
     end
+
+    it "escapes fields with dangerous strings" do
```

Comment:
Minor thing, but can we be more specific in this spec line and maybe mention we want to escape characters that could be dangerous when interpreted by Excel? I know that information is in the commit message, but capturing it here in the spec description makes it easier for a developer to understand why these strings are dangerous.

Reply:
Done.